apps/client/src/components/molecules/button/Button.component.tsx

```diff
@@ -0,0 +1,72 @@
+import { ReactElement } from 'react';
+import { Text } from '../../';
+import { ReactComponent as CarIcon } from '../../../assets/cars/pixelCar.svg';
+import cs from 'classnames';
+
+export interface ButtonProps {
+  onClick: () => void;
+  isCarVisible?: boolean;
+  isDisable?: boolean;
+  isButtonDanger?: boolean;
+  children: string;
+}
+
+export function Button({
+  onClick,
+  isCarVisible,
+  isDisable,
+  isButtonDanger,
+  children,
+}: ButtonProps): ReactElement {
+  isCarVisible ||= false;
+  isDisable ||= false;
+  isButtonDanger ||= false;
+
+  return (
+    <button
+      type='button'
+      onClick={onClick}
+      className={
+        isDisable
+          ? cs(
+              'opacity-40',
+              'cursor-not-allowed',
+              'bg-transparent',
+              'border border-neutral-40',
+              'relative py-2 px-4 w-full min-w-min rounded',
+            )
+          : cs(
+              'group',
+              {
+                'bg-primary-40 hover:bg-primary-30 border-primary-50':
+                  isButtonDanger,
+              },
+              { 'bg-transparent hover:bg-neutral-20': !isButtonDanger },
+              'border hover:border-4 border-neutral-40',
```

Comment:
height of the button changes with the addition of the border on hover. which can cause layout shifts & bad UX. shall we have a transparent border in default state. ?

Reply:
Fixed in PR https://github.com/the-ai-team/razor/pull/47
Storybook Preview https://63d9bf20a85a7e841a116e4f-lpxvcjsmxk.chromatic.com/